interface-definitions/container.xml.in

```diff
@@ -71,6 +71,36 @@
               <multi/>
             </properties>
           </leafNode>
+          <node name="sysctl">
+            <properties>
+              <help>Configure namespaced kernel parameters of the container</help>
+              <priority>318</priority>
```

Comment:
the container has priority 450
```
vyos@r4# /usr/libexec/vyos/priority.py | grep contain
       450  container.py                        ['container']

```
Do we really need priority here?

Reply:
I don't know this functionality of the config definition. I accidentally copied that over from system sysctl.
I can remove it.

Reply:
The priority option shows which component will start/apply first during load config/commit
For example, if we have a tunnel and configuration address for eth interface, firstly, we should assign the address to the ethX interfaces, and then you can use it in the tunnel.
```
vyos@r4# /usr/libexec/vyos/priority.py | match "eth|tun"
       300  interfaces_virtual-ethernet.py      ['interfaces', 'virtual-ethernet']
       318  interfaces_ethernet.py              ['interfaces', 'ethernet']
       321  interfaces_pseudo-ethernet.py       ['interfaces', 'pseudo-ethernet']
       380  interfaces_tunnel.py                ['interfaces', 'tunnel']
[edit]
vyos@r4# 
```
Delete the priority here, we do not need it in this case.

Reply:
thanks, for the explanation, removed it.